downstream/modules/platform/proc-gw-editing-a-user.adoc

```diff
@@ -6,7 +6,7 @@
 
 You can modify the properties of a user account after it is created.
 
-In upgrade scenarios, there might be pre-existing user accounts from {ControllerName}, {EDAName} or {HubName} services. When editing these user accounts, the *User type* checkboxes indicate whether the account had service level administrator privileges. You can revoke or assign administrator permissions for the individual services and designate the user as either an {PlatformNameShort} Administrator, {PlatformNameShort} Auditor or Normal User. Assigning administrator privileges to all of the individual services automatically designates the user as an {PlatformNameShort} Administrator. See xref:proc-controller-creating-a-user[Creating a user] for more information about user types.
+In upgrade scenarios, there might be pre-existing user accounts from {ControllerName} or {HubName} services. When editing these user accounts, the *User type* checkboxes indicate whether the account had service level administrator privileges. You can revoke or assign administrator permissions for the individual services and designate the user as either an {PlatformNameShort} Administrator, {PlatformNameShort} Auditor or Normal User. Assigning administrator privileges to all of the individual services automatically designates the user as an {PlatformNameShort} Administrator. See xref:proc-controller-creating-a-user[Creating a user] for more information about user types.
```

Comment:
@dcdacosta : This is not related to the actual edit you made, but it's in the same para. so feel free to take or leave this comment. Are these UI elements: "{PlatformNameShort} Administrator, {PlatformNameShort} Auditor or Normal User" ? Then it would make sense to have 'admin', 'auditor', and 'normal user' in title case, but also be highlighted in bold formatting. If this means making updates to larger sections within the guide, feel free to reject this comment.